R/components.R

```diff
@@ -58,6 +48,55 @@ input <- function(name, defaultValue) {
   }
 }
 
+select <- function(name) {
+  function(
+    inputId,
+    items,
+    selected = NULL,
+    noResults = "No results.",
+    ...
+    ) {
```

Comment:
1. Indentation.
2. Move `noResults` behind `...` so it needs to be passed by name. In general, let's limit the number of accepted positional arguments - it will make user's code more readable and it will make interface changes easier in the future.

```suggestion
    ...,
    noResults = "No results.",
  ) {
```

Reply:
Both fixed